crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -389,6 +389,14 @@ def prepare_nodes_for_os_upgrade
     end
 
     commit_and_check_proposal
+
+    # FIXME: upgrade of the nodes needs to be correctly orchestrated, to upgrade nodes one by one
+    # Now we're adding this only to imitate original autoyast-based upgrade and to
+    # have current upgrade CI job working
+    upgrade_nodes.each do |node|
+      Rails.logger.info("initiating upgrade of node #{node.name}")
+      node.ssh_cmd("/usr/sbin/crowbar-upgrade-os.sh")
```

Comment:
No, because it executes zypper dup in background. Later, when we use it properly, we have to use some checks if upgrade goes well, but this is really just a temporary hotfix